torch/nn/quantized/modules/conv.py

```diff
@@ -64,7 +64,7 @@ def weight(self):
 
     @weight.setter
     def weight(self, w):
-        self._packed_weight = torch.ops.quantized.fbgemm_conv_prepack(w, self.groups)
+        self._packed_weight = torch.ops.quantized.fbgemm_conv_prepack(w, self.stride, self.padding, self.dilation, self.groups)
```

Comment:
an unrelated question: do we have unpack for conv?

Reply:
@jerryzh168 
We made sure to add unpack for conv. Details in https://github.com/pytorch/FBGEMM/pull/103, https://github.com/pytorch/FBGEMM/pull/105, https://github.com/pytorch/FBGEMM/pull/106, https://github.com/pytorch/FBGEMM/commit/815139b1ba72ed43571084f58c7fdf4ca3c991d5, https://github.com/pytorch/FBGEMM/commit/64a2c73a425e4113839f2c2b596ea28d632d20f6

Reply:
is this ready or still ongoing?

Reply:
oh I saw qconv_unpack, Thanks

Reply:
This is ready. I might have to change your qat modules if they have landed.